docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -348,7 +352,7 @@ role ``meta/argument_specs.yml`` file. All fields are lowercase.
 
             * If ``required`` is ``false``/missing, ``default`` may be specified (assumed ``null`` if missing).
             * Ensure that the default value in the docs matches the default value in the code. The actual
-              default for the role variable will always come from ``defaults/main.yml``.
+              default for the role variable will always come from ``defaults/main.yml`` or ``defaults/main/\*``.
```

Comment:
it is main/main.yml/main.yaml or 'specified' .. probably a section just explaining how this works and precedence

Reply:
I'm struggling a bit with finding out how the user should specify the custom main file. The closest explanation I found is in this section https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html#role-directory-structure where the documentation mentions adding other YAML files in some directories, but I think it's something else, as these file already have the .yml extension. I assume that the explanation I am looking for already exists, and I just want to add a link to that after I explain the precedence. Could you please help me?